PocketKit/Sources/PocketKit/Home/SwiftUI/HomeSlateView.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Sync
+import SwiftData
+
+struct HomeSlateView: View {
+    let remoteID: String
+    let slateTitle: String
+    @Query var recommendations: [Recommendation]
+
+    init(remoteID: String, slateTitle: String) {
+        self.remoteID = remoteID
+        self.slateTitle = slateTitle
+        let predicate = #Predicate<Recommendation> {
+            $0.slate?.remoteID == remoteID
+        }
+        _recommendations = Query(filter: predicate, sort: \Recommendation.sortIndex, order: .forward)
+    }
+    var body: some View {
+        VStack(alignment: .leading, spacing: 16) {
+            // TODO: replace with section header
+            Text(slateTitle)
+            if let item = recommendations.first?.item {
+                HomeHeroView(model: HomeItemCellViewModel2(item: item, imageURL: item.topImageURL))
+            }
+//            List(recommendations) { recommendation in
```

Comment:
why is this commented out?

Reply:
Good catch! There's a lot of temporary stuff before the feature branch gets merged into `develop`.  it's mostly there to remind me that this needs another view in there, but I can convert it to a TODO if you'd prefer